test/cmd/packet-verifier/packet-verifier.go

```diff
@@ -0,0 +1,272 @@
+package main
+
+import (
+	"errors"
+	"flag"
+	"fmt"
+	"net"
+	"os"
+	"time"
+
+	"github.com/google/gopacket"
+	"github.com/google/gopacket/layers"
+	"github.com/google/gopacket/pcap"
+	"github.com/vishvananda/netlink"
+)
+
+const (
+	shortDescription = "packet-verifier"
+	longDescription  = "Packet verifier fetches corresponding interfaces and validates the packets."
+)
+
+var (
+	// version string populated during build.
+	version = "unknown"
+
+	// ip to monitor on the interfaces
+	ipToMonitor string
+	receiverIP  string
+	device      string
+
+	// vlan ID to monitor on the interfaces
+	vlanIDToMonitor int
+
+	// pcap parameters (requires libpcap-devel to be installed on the host)
+	snapshot_len int32 = 1024
+	promiscuous        = false
+	timeout            = 30 * time.Second
+	handle       *pcap.Handle
```

Comment:
Can we move the non-flag variables into `main()`?

In particular `enis` `handle` are probably going to be mutated at runtime and would be easier to think about if they were more narrowly scoped.